File: admin/src/main/java/com/jigumulmi/place/repository/AdminSubwayStationPlaceRepository.java
package com.jigumulmi.place.repository;

import com.jigumulmi.place.domain.Place;
import com.jigumulmi.place.domain.SubwayStationPlace;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AdminSubwayStationPlaceRepository extends JpaRepository<SubwayStationPlace, Long> {

    void deleteAllByPlace(Place place);
}

File: admin/src/main/kotlin/com/jigumulmi/place/repository/AdminSubwayStationPlaceRepository.kt
package com.jigumulmi.place.repository

import com.jigumulmi.place.domain.Place
import com.jigumulmi.place.domain.SubwayStationPlace
import org.springframework.data.jpa.repository.JpaRepository

interface AdminSubwayStationPlaceRepository : JpaRepository<SubwayStationPlace?, Long?> {
    fun deleteAllByPlace(place: Place?)
}
